daffodil-schematron/README.md

```diff
@@ -0,0 +1,48 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+Schematron Validation
+===
+
+Daffodil Validator for Schematron
+
+XSLT implementation adapted from the [Camel Schematron Component](https://github.com/apache/camel/tree/master/components/camel-schematron).
```

Comment:
Adapted in what way?
There are files below that are included (such as a readme.txt) that may not be meaningful in Daffodil context. Were they included by mistake or is there a "include everything" approach here where we're only modifying the tree from Camel in minimal ways so as, for example, to facilitate diffs/merges of fixes etc. from Camel to here, or vice versa?

Reply:
> Adapted in what way?

Adapted as in it was a java implementation that provided the blueprint for loading and constructing the xslt framework.  I converted it to a pure scala implementation.  So maybe "converted" is better than "adapted", I dont know.

The gist of it is I wanted to be sure to point credit back there because I definitely didnt dig into the weeds to understand all of the xslt involved, really just applied heaping amount of scala sugar to their logic.  Which ended up restructuring it quite a bit, but still.

> There are files below that are included (such as a readme.txt) that may not be meaningful in Daffodil context. Were they included by mistake or is there a "include everything" approach

Noted this above in a comment.  I didnt think too hard about it, but I think we can remove them safely after looking over things.

>  .. here where we're only modifying the tree from Camel in minimal ways so as, for example, to facilitate diffs/merges of fixes etc. from Camel to here, or vice versa?

No the "adaptation" here is one-way one-time, any fixes we would notice from their work would be manual ports to ours.

Reply:
So please add these points to the README.md so that we have captured that this is a scala adaptation.

Probably worth it to capture in here also the exact git hash of the Camel version you took this from, that way we can robustly diff to see if fixes have happened to it, and apply them.

Reply:
> So maybe "converted" is better than "adapted", I dont know

I avoided that wordsmithing all together in the latest.

`Scala implementation based on the [Camel Schematron Component]...`

Reply:
> Probably worth it to capture in here also the exact git hash of the Camel version you took this from, that way we can robustly diff to see if fixes have happened to it, and apply them.

yep good call.  done.